pkgs/development/tools/gocyclo/default.nix

```diff
@@ -1,23 +1,20 @@
-{ buildGoPackage
+{ buildGoModule
 , lib
 , fetchFromGitHub
 }:
-
-buildGoPackage rec {
-  pname = "gocyclo-unstable";
-  version = "2015-02-08";
-  rev = "aa8f8b160214d8dfccfe3e17e578dd0fcc6fede7";
-
-  goPackagePath = "github.com/alecthomas/gocyclo";
+buildGoModule rec {
+  pname = "gocyclo";
+  version = "0.4.0";
 
   src = fetchFromGitHub {
-    inherit rev;
-
-    owner = "alecthomas";
+    owner = "fzipp";
     repo = "gocyclo";
-    sha256 = "094rj97q38j53lmn2scshrg8kws8c542yq5apih1ahm9wdkv8pxr";
+    rev = "v${version}";
+    sha256 = "1s9m5m5p76wcxi5n4diz891kd5db4ll21fsh9fnvvf9w7yrmgdw2";
   };
 
+  vendorSha256 = "0sjjj9z1dhilhpc8pq4154czrb79z9cm044jvn75kxcjv6v5l2m5";
+
   meta = with lib; {
     description = "Calculate cyclomatic complexities of functions in Go source code";
     homepage = "https://github.com/alecthomas/gocyclo";
```

Comment:
```suggestion
    homepage = "https://github.com/fzipp/gocyclo";
```

Reply:
Oh fiddlesticks, I fixed that originally, but must've force-pushed over it...

Reply:
Fixed